src/scores.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+
+function compareValues({ player1Card, player2Card }) {
+  return player1Card.value - player2Card.value;
+}
+
+function pointGiver(result, { player1Score, player2Score }) {
+  if (result > 0) {
+    console.log('Player1 scores a point!');
```

Comment:
a ton of redundant code. you should have function printing who won the battle and reuse it with various parameters instead of using console.log 3/4 times

Reply:
fixed